frontend/src/container/Login.jsx

```diff
@@ -151,6 +154,15 @@ class Login extends React.Component {
     if (fetchGetWorkspaceList.status === 200) props.dispatch(setWorkspaceList(fetchGetWorkspaceList.json))
   }
 
+  setDefaultUserLang = async loggedUser => {
+    const { props } = this
+    const fetchPutUserLang = await props.dispatch(putUserLang(loggedUser, props.user.lang))
+    switch (fetchPutUserLang.status) {
+      case 200: break
+      default: props.dispatch(newFlashMessage(props.t('Error while saving your default language')))
```

Comment:
'Error while saving your default language' => 'Error while saving your language' (there is no "default language" for a user, there is a language)

Reply:
By the way, this new message does not seems to have associated translation in french.

Reply:
fixed